ditto/models/position.py

```diff
@@ -3,11 +3,15 @@
 
 from .base import DiTToHasTraits, Float, Unicode, Any, Int, List, observe, Instance
 
+from ..constant import M
+
 
 class Position(DiTToHasTraits):
-    long = Float(help="""Decimal Longitude""")
-    lat = Float(help="""Decimal Latitude""")
-    elevation = Float(help="""Decimal elevation (meters)""")
+    # NOTE: We are not really using lat/long I believe, but mostly coordinates in various format
+    # TODO: Build the conversion to parse to lat/long from given inputs...
```

Comment:
Re unit lat/long conversion, I've used [pyproj](https://pypi.org/project/pyproj/) in the past and it worked well. The only downside is that if one forgets to set ` preserve_units=True` during initialization things silently break when the input is not in metres.